app/inc-category/edit/controller.js

```diff
@@ -36,7 +36,7 @@ export default AbstractEditController.extend({
 
     showDeleteItem(item) {
       let i18n = get(this, 'i18n');
-      let message = i18n.t('incident.messages.deleteItem');
+      let message = i18n.t('messages.delete_singular', { name: 'item' });
```

Comment:
This code is passing a non localized string when it should be passing in a localized string or it should use the name of the item being deleted.